docs/faq.txt

```diff
@@ -132,4 +132,17 @@ Determined has support for TensorFlow models that use the :ref:`tf.keras <tf-ker
 or :ref:`Estimator <estimator-trial>` APIs. For models that use the low-level TensorFlow
 Core APIs, we recommend porting your model to use :ref:`Estimator Trial <estimator-trial>`.
 `Example of converting a Tensorflow graph into an Estimator
-<https://github.com/determined-ai/determined/blob/master/examples/experimental/mnist_tp_to_estimator/model_def.py>`_.
+<https://github.com/determined-ai/determined/blob/master/examples/experimental/trial/mnist_tp_to_estimator/model_def.py>`_.
+
+
+Tensorboard Support
+-------------------
+
+Can I log Tensorboards events in Determined?
```

Comment:
I'm a little confused by how this is worded: isn't all TensorBoard metadata captured as a series of events? Is this talking about logging additional custom data to TensorBoard? I wonder if a code example would be helpful -- i.e., where should the path `/tmp/tensorboard` be passed to?

Reply:
I rephrased this and added a code snipped showing how this can be done for PyTorch.